crates/bevy_core_pipeline/src/upscaling/node.rs

```diff
@@ -84,6 +84,20 @@ impl ViewNode for UpscalingNode {
             .command_encoder()
             .begin_render_pass(&pass_descriptor);
 
+
+        if let Some(camera) = camera {
+            if let Some(viewport) = &camera.viewport {
+                let size =viewport.physical_size;
+                let position = viewport.physical_position;
+                render_pass.set_scissor_rect(
+                    position.x as u32,
+                    position.y as u32,
+                    size.x as u32,
+                    size.y as u32,
+                );
+            }
+        }
```

Comment:
I think this should fix the CI errors.

```suggestion
        if let Some(camera) = camera {
            if let Some(viewport) = &camera.viewport {
                let size = viewport.physical_size;
                let position = viewport.physical_position;
                render_pass.set_scissor_rect(
                    position.x,
                    position.y,
                    size.x,
                    size.y,
                );
            }
        }
```